### DISCUSSION TITLE
Is patient registration an encounter?

### DISCUSSION THREAD

**Discussion by f-odhiambo** [opensrp/fhircore#487] [Ideas]
How do we classify a registration event? Is this an encounter? 

A family visit can be classified as an encounter but registering the family can be a description in the encounter. Open to more views here

**Comment by pld** [comment]
I'd think so, at least conceptually.

In existing OpenSRP apps, is registration counted as an encounter? And why/why not?

**Comment by f-odhiambo** [comment]
Ok. That would mean will have multiple resources creation for the following also 
-  registration we get patient, related person and registration encounter for ANC
- Immunization we create an immunization resource plus related encounter to track that event together with an ImmunizationRecommendation for the next set of dose(s)
